### ISSUE TITLE
Optimize drawing tiles

### ISSUE THREAD

**Issue by kuroneko1996** [kuroneko1996/cyberlab#31]
Static tiles shouldn't be drawn one by one.

**Comment by slemonide** [comment]
So, does it mean we would store the whole map as one picture?

**Comment by kuroneko1996** [comment]
No, it's more about creating one big surface of the static tiles at the loading time and moving it with the camera and drawing.

**Comment by kuroneko1996** [comment]
The image with tiles could be another solution for static tiles, though.
Dynamic tiles should use some batch system.

**Comment by slemonide** [comment]
Should this really be our goal for Demo 2?
At this point, it's not really a performance issue that static tiles are drown separately.
